.github/workflows/codeql_security_checks.yml

```diff
@@ -12,16 +12,26 @@ jobs:
       runs-on: ubuntu-latest
       steps:
         - uses: actions/checkout@v2
-        - uses: github/codeql-action/init@v3
+        - name: Install CodeQL
+          run: |
+            cd ../
+            wget https://github.com/github/codeql-cli-binaries/releases/download/v2.17.2/codeql-linux64.zip
```

Comment:
can we use the link to latest instead of a hardcoded version ? .. https://github.com/github/codeql-cli-binaries/releases/latest/download/codeql-linux64.zip docs; https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases

Reply:
That's a good idea, let's do that next.